_locales/en/messages.json

```diff
@@ -200,6 +200,47 @@
 	"explore_finished": {
 		"message": "Queue has been explored."
 	},
+	"explore_saleitem_trying_to_claim": {
+		"message": "Trying to claim a sale item if there is one…"
+	},
+	"explore_saleitem_cant_claim": {
+		"message": "No sale item to claim at this moment."
+	},
+	"explore_saleitem_next_item_time": {
+		"message": "Next item available: $datetime$",
+		"placeholders": {
+			"datetime": {
+				"content": "$1",
+				"example": "01.07.2024, 20:00:00",
+				"description": "Date and time format depends on locale set in browser"
+			}
+		}
+	},
+	"explore_saleitem_claim_failed": {
+		"message": "Failed to get a sale item after $retries$ retries. Error: $error$",
+		"placeholders": {
+			"retries": {
+				"content": "$1",
+				"example": "5"
+			},
+			"error": {
+				"content": "$2",
+				"example": "HTTP 403"
+			}
+		}
+	},
+	"explore_saleitem_empty": {
+		"message": "No item ID returned while trying to get a sale item, perhaps it was already claimed."
```

Comment:
People gonna be confused by this message, and it generally should not happen due to can claim check. Just use the failed to claim msg.